packages/twenty-front/src/utils/array/recordFromArrayWithValue.ts

```diff
@@ -0,0 +1,10 @@
+// const tmp =  ['foo', 'bar'] as const;
+// const result = recordFromArrayWithValue(tmp, true);
+// returns { foo: true, bar: true }
+// result has strictly typed keys foo and bar
+
+export const recordFromArrayWithValue = <T, U extends string>(
```

Comment:
`buildRecordFromKeysWithSameValue` would be more self-explanatory and begins with a verb, which is what all functions should begin with. Also not really sure that we should create this function but why not.

Reply:
Will update the name !

Needed in order to transpile from an array of object metadata fields name to a `recordGqlFields`